src/containers/AlbumsContainer.jsx

```diff
@@ -1,3 +1,33 @@
 import React, { useState, useEffect } from 'rect';
 import Albums from '../components/Albums/Albums';
-import { fetchArtist }
\ No newline at end of file
+import { fetchArtist fetchAlbum } from '../services/ApiFetch';
+import PropTypes from 'prop-types';
+
+const AlbumsContainer = ({ match }) => {
+    const [ artist, setArtist ] = useState({});
+    const [albums, setAlbums ] = useState([]);
+
+    useEffect(() => {
+        fetchArtist(match.params.id)
+        .then(artist => setArtist(artist));
+
```

Comment:
also i don't think you need to fetch your artist here because you have access to the artist id in your params (match.params.id) — you can set it to a variable and pass it to your Albums component. 
Like...
  const newArtist = match.params.id;

    <Albums artist={newArtist} albums={albums} onBrokenImage={handleNoImage} />